src/foundry/common/data/data.mjs/rollTableData.d.ts

```diff
@@ -0,0 +1,172 @@
+   */
+  img: string;
+
+  /**
+   * The HTML text description for this RollTable document
+   */
+  description: string;
+
+  /**
+   * A Collection of TableResult embedded documents which belong to this RollTable
+   */
+  results: EmbeddedCollection<ConfiguredDocumentClass<typeof documents.BaseTableResult>, RollTableData>;
+
+  /**
+   * The Roll formula which determines the results chosen from the table
+   */
+  formula: string;
+
+  /**
+   * Are results from this table drawn with replacement?
+   * @defaultValue `true`
+   */
+  replacement: boolean;
+
+  /**
+   * Is the Roll result used to draw from this RollTable displayed in chat?
+   * @defaultValue `true`
+   */
+  displayRoll: boolean;
+
+  /**
+   * The _id of a Folder which contains this RollTable
+   */
+  folder: string | null;
+
+  /**
+   * The numeric sort value which orders this RollTable relative to its siblings
+   */
+  sort: number;
+
+  /**
+   * An object which configures user permissions to this RollTable
+   */
+  permission: object;
+
+  /**
+   * An object of optional key/value flags
+   * @defaultValue `{}`
+   */
+  flags: ConfiguredFlags<'RollTable'>;
+}
+
+interface RollTableDataConstructorData {
+  /**
+   * The _id which uniquely identifies this RollTable document
+   */
+  _id?: string;
+
+  /**
+   * The name of this RollTable
+   */
+  name: string;
+
+  /**
+   * An image file path which provides the thumbnail artwork for this RollTable
+   */
+  img?: string;
+
+  /**
+   * The HTML text description for this RollTable document
+   */
+  description?: string;
+
+  /**
+   * A Collection of TableResult embedded documents which belong to this RollTable
+   */
+  results?: ConstructorParameters<ConfiguredDocumentClass<typeof documents.BaseTableResult>>[0][] | null;
+
+  /**
+   * The Roll formula which determines the results chosen from the table
+   */
+  formula?: string;
+
+  /**
+   * Are results from this table drawn with replacement?
+   * @defaultValue `true`
+   */
+  replacement?: boolean;
+
+  /**
+   * Is the Roll result used to draw from this RollTable displayed in chat?
+   * @defaultValue `true`
+   */
+  displayRoll?: boolean;
+
+  /**
+   * The _id of a Folder which contains this RollTable
+   */
+  folder?: string | null;
+
+  /**
+   * The numeric sort value which orders this RollTable relative to its siblings
+   */
+  sort?: number;
+
+  /**
+   * An object which configures user permissions to this RollTable
+   */
+  permission?: object;
+
+  /**
+   * An object of optional key/value flags
+   * @defaultValue `{}`
+   */
+  flags?: ConfiguredFlags<'RollTable'>;
+}
+
+/**
+ * The data schema for an RollTable document.
+ * @see BaseRollTable
+ */
+export declare class RollTableData extends DocumentData<
+  RollTableDataSchema,
+  RollTableDataProperties,
+  PropertiesToSource<RollTableDataProperties>,
+  RollTableDataConstructorData,
+  documents.BaseRollTable
+> {
+  static defineSchema(): RollTableDataSchema;
+}
```

Comment:
It is used, side bar icon, plus on the FormApplication. Documentation error in Foundry. I'll correct the documentation in our copy, since it's a minor one unless you think it's best to no, and file a issue on Foundry.

![image](https://user-images.githubusercontent.com/590498/128608362-aac6bf04-8577-4ebd-89f4-1f2b590b1550.png)